src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -268,6 +276,13 @@ function EVMExecutor({ evmcrispr }) {
 
   return (
     <Box heading="App selector">
+      <Field label="Context">
```

Comment:
What do you think of including a hint with a message suggesting that could be a forum post link?

Reply:
sure